templates/master.yaml.tmpl

```diff
@@ -300,6 +300,9 @@ storage:
                       - mountPath: /lib/modules
                         name: lib-modules
                         readOnly: true
+                      - mountPath: /run/xtables.lock
+                        name: xtables-lock
+                        readOnly: false
```

Comment:
isnt this file? if you try mount file that is not there then docker will create folder instead of it
AFAIK lock files are created on demand so after boot the file wont be there

Reply:
That is a mount. Volume handles this like few rows below

Reply:
I didn’t see lock errors in kube-proxy